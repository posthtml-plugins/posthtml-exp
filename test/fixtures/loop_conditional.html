<p>x</p>
<ul>
  <each loop='number in items'>
    <if condition="number % 2 === 0">
      <li>{{ number }} % 2 === 0</li>
    </if>
    <else>
      <li>{{ number }}</li>
    </else>
  </each>
</ul>
<p>x</p>
