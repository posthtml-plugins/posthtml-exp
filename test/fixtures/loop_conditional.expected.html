<p>x</p>
<ul>


      <li>1</li>


      <li>2 % 2 === 0</li>


      <li>3</li>

</ul>
<p>x</p>
